test/inductor/test_torchinductor.py

```diff
@@ -1745,6 +1745,38 @@ def fn(a, b):
             check_lowp=False,
         )
 
+    @config.patch(use_mixed_mm=True)
+    def test_mixed_mm(self):
+        def fn(a, b):
+            return torch.mm(a, b.to(a.dtype))
+
+        if self.device == "cuda":
```

Comment:
We should test CPU

Reply:
ok, i'll add that, but i think it just errors iirc